package.json

```diff
@@ -22,7 +22,6 @@
     "typescript": "5.4.2"
   },
   "engines": {
-    "node": "18.x",
-    "pnpm": "8.x"
```

Comment:
Works perfectly fine with the latest pnpm version, so removed this to avoid further usage headaches.